src/flag_gems/runtime/register.py

```diff
@@ -0,0 +1,96 @@
+from typing import Optional
+
+import torch
+
+from . import backend, commom_utils, error
+from .device import deviceDetector
```

Comment:
```suggestion
from .device import DeviceDetector
```
Use the same name style?

Reply:
Get